### ISSUE TITLE
Wrong attachment type after chat import

### ISSUE THREAD

**Issue by Fedir-Yatsenko** [epam/ai-dial-chat#1759]
### EPAM AI DIAL chat version

0.11.0

### What happened?

I have a chat with a custom attachment like below:
```
"custom_content": {
	"attachments": [{
		"index": 2,
		"type": "application/dial-ttyd-table",
		"title": "Custom Table",
		"url": "files/.../7-5e676d1d-c2d4-4771-9841-817f8ffbde15.json"
	}]
}
```

I export this chat with attachments, then I import it. In the new chat, this attachment type and name are different:
```
"custom_content": {
	"attachments": [{
		"index": 2,
		"type": "application/json",
		"title": "7-5e676d1d-c2d4-4771-9841-817f8ffbde15.json",
		"url": "files/.../7-5e676d1d-c2d4-4771-9841-817f8ffbde15.json"
	}]
}
```

### Confidential information

- [X] I confirm that do not share any confidential information

**Comment by VolhaBazhkova** [comment]
Unfortunately, on our servers there is no custom renderer configuration. The fix was verified on staging env using the import file provided by separate team (0.15.0 version).  The fix was included into 0.14.1 patch. 
@FedorYatsenko Once your team will install the 0.14.1 build, could you please confirm that everything is correct?

**Comment by Fedir-Yatsenko** [comment]
I checked, everything works, thank you very much!
